LightboxOverlay.js

```diff
@@ -186,7 +186,7 @@ var LightboxOverlay = React.createClass({
 
     var openStyle = [styles.open, {
       left:   openVal.interpolate({inputRange: [0, 1], outputRange: [origin.x, target.x]}),
-      top:    openVal.interpolate({inputRange: [0, 1], outputRange: [origin.y + STATUS_BAR_OFFSET, target.y + STATUS_BAR_OFFSET]}),
+      top:    openVal.interpolate({inputRange: [0, 1], outputRange: [origin.y, target.y]}),
```

Comment:
usually, it's better to change transform style property because it doesn't cause layout calculations and events. So open style after that would look like this: 
```javascript
{
transform: [{
  translateY: openVal.interpolate({inputRange: [0, 1], outputRange: [origin.y, target.y - origin.y]}),
}, {
  translateX: openVal.interpolate({inputRange: [0, 1], outputRange: [origin.x, target.x - origin.x]}),
}]
}
```